examples/tutorials/Part 11b - Secure Classification with Syft Keras and TFE: Secure Model Serving.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>We should change "Served" to be much more informative. Ideally "produced encrypted prediction to &lt;client name&gt;" or at least a list of the servers involved.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>changed, couldn't get a client name unfortunately but it's more informative now</p>